clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -1545,6 +1605,30 @@ bool SemaHLSL::CheckBuiltinFunctionCall(unsigned BuiltinID, CallExpr *TheCall) {
     break;
   }
   case Builtin::BI__builtin_hlsl_elementwise_saturate:
```

Comment:
Did you mean for 'Builtin::BI__builtin_hlsl_elementwise_saturate' to fall through into 'Builtin::BI__builtin_hlsl_select' logic in this switch( ) statement?

Reply:
No I did not; It is supposed to fall through to the next case, so I will reorder these.

Reply:
Sounds good.